src/argilla/client/users.py

```diff
@@ -68,7 +70,7 @@ class User:
     last_name: Optional[str]
     full_name: Optional[str]
     role: UserRole
-    workspaces: Optional[List[str]]
+    workspaces: Optional[List[WorkspaceModel]]
```

Comment:
This should be removed, right?

Reply:
I guess we can keep it as an attribute even though it's a property, but we can also remove it, do you think it's better to remove it?